mungegithub/mungers/bulk-lgtm.go

```diff
@@ -17,12 +17,19 @@ limitations under the License.
 package mungers
 
 import (
+	"bytes"
+	"crypto/rand"
+	"encoding/binary"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"strconv"
+	"strings"
 	"sync"
 
+	"golang.org/x/oauth2"
```

Comment:
It's already there afaik:

https://github.com/kubernetes/test-infra/tree/master/vendor/golang.org/x/oauth2